### PR TITLE
update README and CHANGELOG for 2.8.0

### PR DESCRIPTION
**Pull request by satsukies**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # ChangeLog
 
+## ver 2.8.0
+
+- Support Android Gradle Plugin 8.3.0 (Iguana) with Gradle 8.4
```
Comment (satsukies) [inline CHANGELOG.md:5]: Describe change target strictly.

File: README.md [hunk 2]
```diff
@@ -87,12 +87,13 @@ This plugin does not work with non-app modules and/or library modules correctly.
...
 | 8.1.x (Giraffe)       | \>=2.7.0                     |
 | 8.2.x (Hedgehog)      | \>=2.7.0                     |
+| 8.3.x (Iguana)        | \>=2.8.0                     |
 
 ### Canary/Beta support table
 
 We are not supporting Canary(Alpha) versions because of API stability. Please wait for beta releases or open an issue to support Canary versions.
 
-> As of 2023 Dec 1st, Iguana Canary doesn't work.
+> As of 2024 Mar 8th, Jellyfish is not tested yet.
```
Comment (satsukies) [inline README.md:96]: We have not tested Jellyfish yet, so I updated this sentense.

**Review by jmatsu — APPROVED** [review]
LGTM

**Comment by satsukies** [comment]
Thanks!

**Pull request merged**